src/main/resources/locales/en/crime-rejected-login.json

```diff
@@ -0,0 +1,8 @@
+{
+    "home": "Home",
+    "paragraph1": "You have successfully signed into your Common Component account. Unfortunately, your account role does not allow you to access the verified user part of the Court and tribunal hearings service",
+    "paragraph2": "Solicitors and other legal professionals are the only users allowed to sign in with a Common Component account. For more information about MyHMCTS the link below:",
+    "link": "https://www.gov.uk/guidance/myhmcts-online-case-management-for-legal-professionals",
```

Comment:
MyHMCTS is for CFT only right

Reply:
Thanks Kian, yah I knew this issue. We discussed it after the standup, we do not need this page anymore. We will display generic error message page.